plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -8661,6 +8662,19 @@ protected TestTable createTableWithDefaultColumns()
         throw new SkipException("Hive connector does not support column default values");
     }
 
+    @Override
+    protected OptionalInt maxTableNameLength()
+    {
+        // This value depends on HMS type
```

Comment:
HMS is Hive metastore, so it's a concrete type

```suggestion
        // This value depends on metastore type
```